tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
It's in [the spec](https://facebook.github.io/jsx/). It's a `JSXNamespacedName`.